dist/polyfill.object-fit.js

```diff
@@ -36,9 +36,22 @@
 		return items;
 	};
 
+	// get host of stylesheet
+	var getHost = function(href) {
+		var l = document.createElement("a");
```

Comment:
Would be great to not use an abbreviated variable name but sth like `fakeLinkOfSheet`. Also please use single quotes for `createElement`.